source/PreferencesPanel.cpp

```diff
@@ -528,6 +532,16 @@ void PreferencesPanel::DrawSettings()
 		}
 		else if(setting == EXPEND_AMMO)
 			text = Preferences::AmmoUsage();
+		else if(setting == DATE_FORMAT)
+		{
+			Preferences::DateFormat datefmtShortName = Preferences::GetDateFormat();
```

Comment:
You need to set `isOn = true;` here, just like for the settings below. This is also why it uses the disabled color instead of the bright one.

Reply:
> @quyykk: You need to set `isOn = true;` here, just like for the settings below. This is also why it uses the disabled color instead of the bright one.

It causes a compile error.

Reply:
You added it at the wrong place lol

Reply:
Epic fail

Reply:
https://github.com/endless-sky/endless-sky/pull/7324/commits/a59673a900e9fb3707f6d8850e7464e66a74a297

fixed